numpy/ma/core.py

```diff
@@ -4386,12 +4394,12 @@ def get_real(self):
 
         See Also
         --------
-        get_imag, real, imag
+        imag, get_real, get_imag
```

Comment:
Drop the getter functions here too - no one should be using those, so let's not draw attention to them